src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -24,22 +25,66 @@ public class BuildingBlockSwap : IWithName
    {
       public PKSimBuildingBlockType Type { get; set; }
       public string Name { get; set; }
-      public string SnapshotPath { get; set; }
+      public string SnapshotFile { get; set; }
 
-      public void Deconstruct(out PKSimBuildingBlockType type, out string name, out string snapshotPath)
+      public void Deconstruct(out PKSimBuildingBlockType type, out string name, out string snapshotFile)
       {
          type = Type;
          name = Name;
-         snapshotPath = SnapshotPath;
+         snapshotFile = SnapshotFile;
       }
    }
 
-   public class QualifcationConfiguration
+   public class SimulationPlot
    {
-      public string SnapshotPath { get; set; }
+      public string Simulation { get; set; }
+      public int SectionId { get; set; }
+   }
+
+   public class QualifcationConfiguration : IValidatable
+   {
+      /// <summary>
+      ///    Path of project snapshot file used for this qualificaiton run
+      /// </summary>
+      public string SnapshotFile { get; set; }
+
+      /// <summary>
+      ///    Output folder where project artefacts will be exported. It will be created if it does not exist
+      /// </summary>
       public string OutputFolder { get; set; }
-      public string ObservedDataFolderName { get; set; } = CoreConstants.DEFAULT_QUALIFICATION_OBSERVED_DATA_FOLDER_NAME;
+
+      /// <summary>
+      ///    Folder were observed data will be exported
+      /// </summary>
+      public string ObservedDataFolder { get; set; }
+
+      /// <summary>
+      ///    Path of mapping file that will be created for the project.
+      /// </summary>
+      public string MappingFile { get; set; }
+
+      /// <summary>
+      ///    Path of configuration file that will be created as part of the qualificaton run
+      /// </summary>
+      public string ReportConfigurationFile { get; set; }
+
+      public SimulationPlot[] SimulationCharts { get; set; }
+
       public BuildingBlockSwap[] BuildingBlocks { get; set; }
+
+      public IBusinessRuleSet Rules { get; } = new BusinessRuleSet();
+
+      public QualifcationConfiguration()
+      {
+         Rules.AddRange(new[]
```

Comment:
Because we start to have a lot of rules here, I decided to use the rule concept. The runner only have to validate the qualification